### ISSUE TITLE
Direct transfer is more than style conversion accuracy

### ISSUE THREAD

**Issue by Adorablepet** [Simon4Yan/Learning-via-Translation#16]
Environment: Pytorch 0.4 Python 3.5 Titan XP
Problem: I trained directly on the dukemtmc dataset and tested it on the market1501 dataset. MAP:40%, Top1: 66.6%.I trained  on the duk2market dataset and tested it on the market1501 dataset. MAP:22.6%, Top1: 50.1%.I downloaded duke2market on your side.This is somewhat inconsistent with the description of the paper.The reid model is referenced https://github.com/dapengchen123/crf_affinity.Can you help me?Thank you.

**Comment by Simon4Yan** [comment]
When using the pytorch framework [pytorch framework](https://github.com/Simon4Yan/Person_reID_baseline_pytorch) train the re-ID model, we gain 44.3% in rank-1 accuracy on Duke (source dataset is Market). This accuracy is higher than our conference paper.

Moreover, my friends ([huang](https://github.com/huanghoujing) and [Han](https://github.com/GuangxingHan) ) also conduct the experiment on the pytorch framework, of course, they get similar results. Our experience is that the BN layer should be fixed during training (see [here](https://github.com/Simon4Yan/Learning-via-Translation/issues/1)). 

BTW, we will release our lastest reserch on domain adaptation of re-ID in few days (I am in ill now ==).

**Comment by Adorablepet** [comment]
@Simon4Yan Thank you very much for your answer. I hope that you can recover soon and pay attention to rest.
